### PR TITLE
Fix typos by codespell

### PR DESCRIPTION
**Pull request by kianmeng**
# Description

Found via `codespell -S target -L crate,ser,numer,falsy,ro,te,nd,bu,ndoes,statics,ons,fo,rouge,pard`

# User-Facing Changes

None.

# Tests + Formatting

None and done.

# After Submitting

None.

### CODE REVIEW

File: crates/nu-command/src/filters/split_by.rs [hunk 1]
```diff
@@ -21,7 +21,7 @@ impl Command for SplitBy {
     }
 
     fn usage(&self) -> &str {
-        "Create a new table splitted."
+        "Create a new table split."
```
Comment (rgwood) [inline crates/nu-command/src/filters/split_by.rs:24]: I would leave this one as-is. 

"Create a new table split." doesn't make sense, and the `split-by` command is an unusual command that we will probably get rid of eventually.

**Review by rgwood — CHANGES_REQUESTED** [review]
Looks good to me, other than the `split-by` description.

File: crates/nu-command/src/filters/split_by.rs (hunk 1 above)
Comment (fdncred) [reply crates/nu-command/src/filters/split_by.rs:24]: How about `Create a new table that has been split by the passed in column name`?

**Comment by fdncred** [comment]
ya, this is good work and helpful. thanks for the contribution.

**Review by rgwood — APPROVED** [review]

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**